src/OmniSharp.MSBuild/ProjectFile/ProjectFileInfo.ProjectData.cs

```diff
@@ -233,8 +238,13 @@ public static ProjectData Create(MSB.Execution.ProjectInstance projectInstance)
                     {
                         references.Add(fullPath);
                     }
+                    var aliases = referencePathItem.GetMetadataValue("Aliases");
```

Comment:
please add this to https://github.com/OmniSharp/omnisharp-roslyn/blob/master/src/OmniSharp.MSBuild/ProjectFile/MetadataNames.cs

Reply:
add `MetadataNames.Aliases = nameof(Aliases)`, thanks.